lib/src/components/NxBinaryDonutChart/__tests__/NxDonutChart.test.tsx

```diff
@@ -8,12 +8,32 @@ import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxBinaryDonutChart from '../NxBinaryDonutChart';
 
 describe('NxBinaryDonutChart', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, {percent: 0});
+  const getShallowComponent = enzymeUtils.getShallowComponent(NxBinaryDonutChart, {value: 0});
+
+  it('sets the role to "meter"', function() {
+    expect(getShallowComponent()).toHaveProp('role', 'meter');
+  });
+
+  it('sets the aria-valuemin to 0', function() {
+    expect(getShallowComponent()).toHaveProp('aria-valuemin', 0);
+  });
+
+  it('sets the aria-valuemax to 100 when maxVal is not specified', function() {
+    expect(getShallowComponent()).toHaveProp('aria-valuemax', 100);
+  });
+
+  it('sets the aria-valuemax to maxVal when maxVal is specified', function() {
+    expect(getShallowComponent({maxVal: 200})).toHaveProp('aria-valuemax', 200);
+  });
+
+  it('sets the aria-valuenow to value', function() {
+    expect(getShallowComponent({value: 30})).toHaveProp('aria-valuenow', 30);
+  });
 
   it('renders an svg with the expected properties', function() {
-    expect(getShallowComponent({ percent: 90 })).toHaveClassName('.nx-binary-donut-chart');
```

Comment:
Needs some unit tests for `percent`

Reply:
I keep the original percent unit tests and add calculated percent unit tests when value is specified. 99e2bdd95b3f5c95162458b610abb575dbd9ab53